makefile

```diff
@@ -1049,10 +1048,10 @@ $(RELEASEDIR.daal)/$1/$2: \
 	python ./deploy/local/generate_win_solution.py $1 $(RELEASEDIR.daal)/$1 $2 --template_name $3
```

Comment:
i think that entire release VS solution should be removed(1035 - 1055) as well as ./deploy/local/generate_win_solution.py

Reply:
Thanks Nikolay. Fixed.